### ISSUE TITLE
Portable version for Windows?

### ISSUE THREAD

**Issue by GeniusBroccoli** [dynobo/normcap#336]
Are there any plans to make portable builds? Or maybe there is a way to make an installed program portable?

**Comment by dynobo** [comment]
Good idea, @GeniusBroccoli ! As it is quite low priority for me, I'd be happy about contributions to support this!

**Comment by koppor** [comment]
Just as an idea: To do a little step here, maybe one could replace the dependency on Tesseract by using Window's build in OCR functionality (this refs https://github.com/dynobo/normcap/pull/599#issuecomment-1904077278).

**Comment by dynobo** [comment]
@koppor , good idea

The docs, though, seem to indicate, that it might not be possible to use the built-in OCR in portable apps:

> **Remarks**
> APIs in the Windows.Media.Ocr namespace are only supported for desktop apps with package identity. This means that the app is installed and run from an MSIX package. For more information, see Features that require package identity. For information about calling these APIs from a desktop app, see Call Windows Runtime APIs in desktop apps.

Another question I couldn't answer, yet: What's the privacy statement for using Windows OCR? Is it 100% offline? Or are samples being send to the cloud?

**Comment by dynobo** [comment]
A good starting point for working on this issue might be the [briefcase docs](https://briefcase.readthedocs.io/en/stable/reference/platforms/windows/app.html): 

It seems like `briefcase` (the tool used to package NormCap) already supports a `zip` format. 

So, a first step might be to modify [the build script](https://github.com/dynobo/normcap/blob/main/bundle%2Fplatforms%2Fwindows_briefcase.py) to use that option, run [bundle/build.py](https://github.com/dynobo/normcap/blob/main/bundle%2Fbuild.py) and see if something useful drops out.
